engine/menus/main_menu.asm

```diff
@@ -268,7 +268,9 @@ LinkMenu:
 	ld c, 50
 	call DelayFrames
 	ld hl, wd732
-	res 1, [hl]
+	; This tries to clear the debug mode bit when
+	; linking, but it doesn't seem to work as intended.
```

Comment:
So does it work or not? Or works sometimes?

Reply:
It didn't work when I tested. I linked both debug and non-debug builds with `BIT_DEBUG_MODE` set in the save files, but `wd732` is never touched.

Reply:
Well, `ld hl, wd732 / res BIT_DEBUG_MODE, [hl]` absolutely is clearing that bit, so do you mean it gets re-enabled at some point? If so:
```
; This tries to clear the debug mode bit when linking,
; but does not work: if debug mode was on it gets
; re-enabled some time after this.
```
(Be specific about "some time" if you know when it happens.)

Reply:
Also whatever the comment is should go above the pair of lines, and maybe even surround it with blank lines.

Reply:
> so do you mean it gets re-enabled at some point?

Notwithstanding any potential distraction on my part, I observed `wd732` in BGB's debugger immediately before, during, and after linking two players, and its value never changed. It could be that the value did change, but the debugger didn't reflect that for whatever reason (does it only update when the window is focused)?

I'll perform some additional tests.

Reply:
I can confirm it's working correctly so that comment will also be removed. My apologies for the confusion.